dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/com/ibm/ws/cxf/rt/bindings/soap/NoOpActivator.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cxf.rt.bindings.soap;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+/**
+ * This is intended to override the HTTPTransportActivator that
+ * ships from Apache CXF.  We use the Liberty Channel Framework
+ * to handle the underlying transport rather than the HTTP
+ * transport layer provided by CXF.
+ */
+public class NoOpActivator implements BundleActivator {
```

Comment:
Don't bring this class forward, it can prevent the CXF apis from being properly loaded.